### PR TITLE
Client: Make client reassign IDs based on datapackage

### PR DESCRIPTION
**Pull request by silasary**
Brazenly empowered by #10, I started doing more aggressive improvements to an in-progress apworld, which exposed new flaws.

This update syncs the network IDs back into your item and location tables, overwriting the ones generated at load time.  Which prevents issues where inserting a new item halfway down your json file causes everything later in the file to take on the wrong name.

### CODE REVIEW

**Comment by silasary** [comment]
I'm not quite sure why, but Locations don't want to be re-id'd.  I'm guessing we set them up differently?

Regardless, this works as intended.

**Comment by silasary** [comment]
Also, we store the last Game string in persistant storage now, because connecting and disconnecting once a minute was being slowed down by needing to type the game string each time 😛

File: src/ManualClient.py [hunk 1]
```diff
@@ -394,8 +407,9 @@ def update_tracker_and_locations_table(self, update_highlights=False):
                                         # Get the item name from the item Label, minus quantity, then do a lookup for count
                                         old_item_text = item.text
                                         item_name = re.sub("\s\(\d+\)$", "", item.text)
+                                        item_id = self.ctx.item_names_to_id[item_name]
                                         item_data = self.ctx.get_item_by_name(item_name)
```
Comment (FuzzyGamesOn) [inline src/ManualClient.py:411]: item_data looks to not be used anymore after adding the item_id var.

**Review by FuzzyGamesOn — CHANGES_REQUESTED** [review]

File: src/ManualClient.py (hunk 1 above)
Comment (silasary) [reply src/ManualClient.py]: Ah, yep. My IDE didn't catch it because we reuse the name later in the method.  Fixed.

**Pull request merged**